pkg/rpc/server/server.go

```diff
@@ -2190,3 +2193,12 @@ func validateAddress(addr interface{}) bool {
 	}
 	return false
 }
+
+func escapeForLog(in string) string {
+	return strings.Map(func(c rune) rune {
+		if c < 32 || !strconv.IsGraphic(c) {
```

Comment:
Do we really need a separate case for `c<32`? `strconv.IsGraphic` covers it.

Reply:
Well, it doesn't matter much as CodeQL is still _not_ satisfied with the fix.